site/gatsby-site/README.md

```diff
@@ -18,235 +31,223 @@ Once you have cloned the repository, to set up a local development environment f
-And finally, as part of the site build process, we processed all pending notifications (`processed=false`), send the emails to all recipients, and update the items with `processed=true` and `sentDate=[now]`.
-
-#### Notifications collection definition
-
-- **Incident Updated**
-    ```
-    {
-        "type": "incident-updated",
-        "incident_id": 374,
-        "processed": false
-    }
-    ```
-- **New Incident Report**
-    ```
-    {
-        "type": "new-report-incident",
-        "incident_id": 374,
-        "report_number": 2172,
-        "processed": false
-    }
-    ```
-- **New Incident**
-    ```
-    {
-        "type": "new-incidents",
-        "incident_id": 374,
-        "processed": false
-    }
-    ```
-- **Entities**
-    ```
-    {
-        "type": "entity",
-        "incident_id": 374,
-        "entity_id": "openai",
-        "isUpdate": true,
-        "processed": false
-    }
-    ```
-- **Submission Promoted**
-    ```
-    {
-        "type": "submission-promoted",
-        "incident_id": 374,
-        "processed": false
-    }
-    ```
\ No newline at end of file
+Running this command is only necessary after making changes to the GraphQL schema or queries.
+
+## Additional Configuration
+
+### Using Magic Links for Local Authentication
+
+During local development, you can use the magic links utility script to simulate the authentication flow without setting up email services. This is particularly useful for testing and debugging authentication features.
+
+The script generates magic login links that would normally be sent via email.
+
+Usage:
+
+```bash
+npm run magic-link <email> [callbackUrl] 
+```
+
+Examples:
+
+```
+# Generate a magic link that redirects to homepage after login
+npm run magic-link user@example.com /
+
+# Generate a magic link that redirects to incidents page
+npm run magic-link user@example.com /apps/discover
+
+# Generate a magic link for admin testing
+npm run magic-link admin@example.com /admin/dashboard
+```
+
+### Faster Development Builds
+
+When building the site, some steps can take a while to run. This can be inconvenient when you are working on a feature unrelated to the steps taking the most time in the build process. To avoid this problem, you can set the environment variable `SKIP_PAGE_CREATOR` to a comma-separated list of page-creator functions found in [`gatsby-node`](https://github.com/responsible-ai-collaborative/aiid/blob/main/site/gatsby-site/gatsby-node.js) that should be skipped. These include: `createMdxPages`, `createCitationPages`, `createWordCountsPages`, `createBackupsPage`, `createTaxonomyPages`, `createDownloadIndexPage`, `createDuplicatePages`, `createTsneVisualizationPage`, and `createEntitiesPages`. For instance, to run a development build skipping the creation of the TSNE (spatial) visualization and citation pages, you would run:
```

Comment:
Change `This can be inconvenient when you are working on a feature unrelated to the steps taking the most time in the build process.` to `This can be inconvenient if you're working on a feature that isn't related to the slowest steps in the build process.`
I know if was already like this, but re-reading it that sentence is a bit confusing.